tools/Cargo.lock

```diff
@@ -3769,9 +3769,9 @@ dependencies = [
 
 [[package]]
 name = "webpki"
-version = "0.22.0"
+version = "0.22.1"
```

Comment:
How did this end up affecting tools? The tools Cargo.lock is now gone.

Reply:
It was a dependency of `pubsys` coming from `coldsnap` and `aws-smithy-client`.